packages/browser/src/integrations/breadcrumbs.ts

```diff
@@ -30,21 +31,24 @@ export class Breadcrumbs implements Integration {
   /**
    * @inheritDoc
    */
-  public static id: string = 'Breadcrumbs';
+  public static id: string = BREADCRUMB_INTEGRATION_ID;
 
   /**
    * @inheritDoc
    */
   public name: string = Breadcrumbs.id;
 
-  /** JSDoc */
-  private readonly _options: BreadcrumbsOptions;
+  /**
+   * Options of the breadcrumbs integration.
+   */
+  // This field is public, because we use it in the browser client to check if the `sentry` option is enabled.
+  public readonly options: Readonly<BreadcrumbsOptions>;
```

Comment:
This now means that `options` cannot be minified - but I guess not the end of the world.